setup.py

```diff
@@ -17,15 +17,7 @@
     "Intended Audience :: Developers",
     "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
     "Programming Language :: Python",
-    "Programming Language :: Python :: 2",
-    "Programming Language :: Python :: 2.6",
     "Programming Language :: Python :: 2.7",
```

Comment:
That's great. Now they will instead think "Oh, I wish this was supported by 2.6, or 3.5" and ask for it. This is more manageable and expected.